src/plugins/custom_import_map/opensearch_dashboards.json

```diff
@@ -1,7 +1,7 @@
 {
     "id": "customImportMapDashboards",
-    "version": "2.3.0.0",
-    "opensearchDashboardsVersion": "2.3.0",
+    "version": "2.4.0.0",
+    "opensearchDashboardsVersion": "2.3.1",
```

Comment:
this is interesting, it should also be 2.4.0

Reply:
OpenSearchDashboards does not have 2.4 branch yet. So, cannot set it as 2.4.0 for now.